README.md

```diff
@@ -118,10 +118,11 @@ Summary is exposed as multiple numerical values:
 ```lua
 local metrics = require('metrics')
 
--- create a summary
+-- create a summary with a sliding wingow with 5 age buckets and 600s bucket lifetime
```

Comment:
To be honest, it's pretty hard to understand how exactly sliding window works without explicit explanation in PR description